editor/src/core/shared/template-path.ts

```diff
@@ -262,18 +269,25 @@ export function filterScenes(paths: TemplatePath[]): StaticInstancePath | Instan
 // FIXME: This should be retired, it's just plain dangerous.
```

Comment:
note to self: if we'd move the valid path out of the DOM, we could kill the fromString, that is the only place it is used at!

Reply:
Unless we have to added stringified template paths to the DOM for some sort of upcoming drilling work :grimacing: ...

Reply:
(I'm not saying we do, just that that's a possibility)